cts/cli/crm_mon-T180.xml

```diff
@@ -142,7 +142,7 @@
       <lrm id="httpd-bundle1-0">
         <lrm_resources>
           <lrm_resource id="httpd1" type="apache" class="ocf" provider="heartbeat">
-            <lrm_rsc_op id="httpd1_last_0" operation_key="httpd1_start_0" operation="start" crm-debug-origin="do_update_resource" crm_feature_set="3.0.14" transition-key="33:12:0:173a0661-aaa2-4c3d-a9a3-2e9342720d2d" transition-magic="0:0;33:12:0:173a0661-aaa2-4c3d-a9a3-2e9342720d2d" exit-reason="" on_node="cent7-host1" call-id="8" rc-code="0" op-status="0" interval="0" last-run="1544070233" last-rc-change="1544070233" exec-time="1116" queue-time="0" op-digest="f2317cad3d54cec5d7d7aa7d0bf35cf8"/>
+            <lrm_rsc_op id="httpd1_last_0" operation_key="httpd1_start_0" operation="start" crm-debug-origin="do_update_resource" crm_feature_set="3.0.14" transition-key="33:12:0:173a0661-aaa2-4c3d-a9a3-2e9342720d2d" transition-magic="0:0;33:12:0:173a0661-aaa2-4c3d-a9a3-2e9342720d2d" exit-reason="" on_node="httpd-bundle1-0" call-id="8" rc-code="0" op-status="0" interval="0" last-run="1544070233" last-rc-change="1544070233" exec-time="1116" queue-time="0" op-digest="f2317cad3d54cec5d7d7aa7d0bf35cf8"/>
```

Comment:
It's strange that the on_node here was `cent7-host1` in the first place.

(I've only spot-checked these tests, due to the sheer number, and noticed this only now.)

Reply:
That does look strange. I'm not sure why it's like that, but we should add on_node only if it doesn't already exist.